api/v1/common_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022.
+Copyright 2024.
```

Comment:
Is it necessary to change the Copyright year?

Reply:
Found https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated, So I think we should change it to 2025?

Reply:
It's actually a new file, but git thinks it's a rename of `api/catalogd/doc.go`...
And you're right, it ought to be 2025

Reply:
We don't have a copyright policy... something we may want to talk about in one of our meetings.